db/database.py

```
@@ -0,0 +1,537 @@
+    
+    @classmethod
+    # TODO: chenge return type to IdentityKeyPair
+    def get_signed_pre_key_pair(
+        cls,
+        _session: Session,
+        uuid: str,
+        device_id: int = 1,
+        with_private: bool = True,
+    ) -> Union[Dict[str, str], None]:
+        return cls._get_key_pair(_session, uuid, device_id, "fake_signed_pre_key", with_private)
+        return cls._get_key_from_list(_session, key_type, aci, device_id, "fake_signed_pre_key", key_id, with_private)
```

Ruff

Undefined name `key_type`

(F821)